dll/win32/hhctrl.ocx/chm.c

```diff
@@ -459,7 +460,7 @@ BOOL LoadWinTypeFromCHM(HHInfo *info)
     /* merge the new data with any pre-existing HH_WINTYPE structure */
     MergeChmProperties(&wintype, info, FALSE);
     if (!info->WinType.pszCaption)
-        info->WinType.pszCaption = info->stringsW.pszCaption = strdupW(info->pCHMInfo->defTitle ? info->pCHMInfo->defTitle : empty);
+        info->WinType.pszCaption = info->stringsW.pszCaption = strdupW(info->pCHMInfo->defTitle ? info->pCHMInfo->defTitle : def_titleW);
```

Comment:
Can just put `L"Help"` here directly

Reply:
I just follow the original coding style.

Reply:
I just follow the original coding style.

Reply:
I use the original coding style, for readability.

Reply:
Why don't you add the string to the language resource files?

Reply:
> Why don't you add the string to the language resource files?

That's a good idea.

Reply:
> Why don't you add the string to the language resource files?

Oh, I forgot about the translations. 😄